### PR TITLE
Video page

### PR DESCRIPTION
**Pull request by mikesch803**
1. add single video page
2. integrate GET /api/video/:videoId api
3.  readded sign up page due to netlify deploy issue

# files to check
- [ ] src/context/video-context.jsx
- [ ] src/pages/home-page/Home.jsx
- [ ] src/pages/video-listing-page/VideoListing.jsx
- [ ] src/pages/video-page/Video.jsx
- [ ] src/reducer/VideoReducer.jsx

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,8 +23,9 @@ function App() {
       <Routes>
         <Route path="/trend" element={<VideoListing />} />
         <Route path="/" element={<Home />} />
+        <Route path="/video" element={<Video/>}/>
         <Route element={<NotAuth />}>
-          <Route path="/signup" element={<Signup />} />
+          {/* <Route path="/signup" element={<Signup />} /> */}
```
Comment (MayankKumar10) [inline src/App.js]: remove this

File: src/reducer/VideoReducer.jsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
 export  const videoReducer = (state, action) => {
     switch (action.type) {
```
Comment (MayankKumar10) [inline src/reducer/VideoReducer.jsx:2]: you can destructure action further by {type, payload}

**Review by foolhardy21 — APPROVED** [review]
`useCategory` is a good use of custom hooks. nothing wrong stands out to me. good to go ahead.

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good to me you can merge it.

**Pull request merged**